File: app/src/main/java/com/worldventures/dreamtrips/social/ui/tripsimages/service/analytics/TripImageVideo360StartedDownloadingAction.java
package com.worldventures.dreamtrips.social.ui.tripsimages.service.analytics;

import com.worldventures.dreamtrips.core.utils.tracksystem.AnalyticsEvent;
import com.worldventures.dreamtrips.core.utils.tracksystem.ApptentiveTracker;
import com.worldventures.dreamtrips.core.utils.tracksystem.BaseAnalyticsAction;

@AnalyticsEvent(action = "membership",
                category = "nav_menu",
                trackers = ApptentiveTracker.TRACKER_KEY)
public class TripImageVideo360StartedDownloadingAction extends BaseAnalyticsAction {
}

File: app/src/main/java/com/worldventures/dreamtrips/social/ui/tripsimages/service/analytics/TripImageVideo360StartedDownloadingAction.kt
package com.worldventures.dreamtrips.social.ui.tripsimages.service.analytics

import com.worldventures.janet.analytics.AnalyticsEvent
import com.worldventures.core.service.analytics.ApptentiveTracker
import com.worldventures.core.service.analytics.BaseAnalyticsAction

@AnalyticsEvent(action = "membership", category = "nav_menu", trackers = arrayOf(ApptentiveTracker.TRACKER_KEY))
class TripImageVideo360StartedDownloadingAction : BaseAnalyticsAction()
